From c87044b9007fa5f6331206b5edd9f7b6bfde4cef Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Dan=20=C4=8Cerm=C3=A1k?= <dcermak@suse.com>
Date: Fri, 22 Jan 2021 09:35:47 +0100
Subject: [PATCH 11/12] Remove not required bcrypt_pbkdf gem

This gem is no longer required by vagrant and was forgotten to get removed.

(cherry picked from commit c9ecc9041a910f192c4498dee045c96e3b8bb6a7)
---
 vagrant.gemspec | 1 -
 1 file changed, 1 deletion(-)

diff --git a/vagrant.gemspec b/vagrant.gemspec
index 3030a63e6..225283771 100644
--- a/vagrant.gemspec
+++ b/vagrant.gemspec
@@ -15,7 +15,6 @@ Gem::Specification.new do |s|
   s.required_ruby_version     = "~> 2.5", "< 2.8"
   s.required_rubygems_version = ">= 1.3.6"
 
-  s.add_dependency "bcrypt_pbkdf", "~> 1.0.0"
   s.add_dependency "childprocess", "~> 4.0.0"
   s.add_dependency "ed25519", "~> 1.2.4"
   s.add_dependency "erubi"
-- 
2.30.0

